sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -311,6 +304,38 @@ public JournalPersonDto getPersonForJournal(String uuid) {
 			return null;
 		}
 	}
+	
+	private String getJournalEmailAddress(PersonDto person) {
+		try {
+			return person.getEmailAddress(false);
+		} catch (PersonDto.SeveralNonPrimaryContactDetailsException e) {
+			logger.error("Error retrieving email address", e);
+			return StringUtils.EMPTY;
+		}
+	}
+	
+	private String getJournalPhoneNumber(PersonDto person) {
+		try {
+			String phoneNumber = person.getPhone(false);
+			if (StringUtils.EMPTY.equals(phoneNumber)) {
+				return phoneNumber;
+			}
+
+			if (configFacade.getPatientDiaryConfig().isActive()) {
+				PhoneNumberUtil phoneUtil = PhoneNumberUtil.getInstance();
+				Phonenumber.PhoneNumber numberProto = phoneUtil.parse(phoneNumber, "DE");
+				return phoneUtil.format(numberProto, PhoneNumberUtil.PhoneNumberFormat.INTERNATIONAL);
+			} else {
+				return phoneNumber;
+			}
+		} catch (PersonDto.SeveralNonPrimaryContactDetailsException e) {
+			logger.error("Error retrieving phone number", e);
```

Comment:
I don't think we should log those as errors. It does refer to an invalid system state and will create further noise in the logs. The probleme here is rather something that should be displayed to the user.

Reply:
For climedo,  this is done already:
https://github.com/hzi-braunschweig/SORMAS-Project/blob/6aeae8e9ac554b09c5dee74920dc01024945d8be/sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java#L367